src/main/java/cohen/maze/mazeGUI/AnimatedMaze.java

```diff
@@ -0,0 +1,85 @@
+package cohen.maze.mazeGUI;
+
+import cohen.maze.Cell;
+import cohen.maze.Maze;
+
+import javax.imageio.ImageIO;
+import javax.swing.*;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+
+
+public class AnimatedMaze extends JComponent {
+    private Maze maze;
+    private Man man;
+    private BufferedImage manImage;
+    private int componentHeight;
+    private int seventhHeight;
+
+
+    public AnimatedMaze(Maze maze, Man man) {
+        this.maze = maze;
+        this.man = man;
+    }
+
+    protected void paintComponent(Graphics g) {
+        //having trouble loading.
+        try {
+            manImage = ImageIO.read(AnimatedMaze.class.getResourceAsStream("/shortMan.png"));
```

Comment:
Do this in the constructor so it only happens once, not in paintComponent() where it happens every time the player moves.